src/View/UIViewTrait.php

```diff
@@ -29,12 +29,14 @@ public function initializeUI(array $options = [])
             $this->layout = $options['layout'];
         }
 
-        $this->loadHelper('Html', ['className' => 'BootstrapUI.Html']);
-        $this->loadHelper('Form', ['className' => 'BootstrapUI.Form']);
-        $this->loadHelper('Flash', ['className' => 'BootstrapUI.Flash']);
-        $this->loadHelper('Paginator', ['className' => 'BootstrapUI.Paginator']);
-        if (class_exists('\Cake\View\Helper\BreadcrumbsHelper')) {
-            $this->loadHelper('Breadcrumbs', ['className' => 'BootstrapUI.Breadcrumbs']);
-        }
+        $helpers = [
+            'Html' => ['className' => 'BootstrapUI.Html'],
+            'Form' => ['className' => 'BootstrapUI.Form'],
+            'Flash' => ['className' => 'BootstrapUI.Flash'],
+            'Paginator' => ['className' => 'BootstrapUI.Paginator'],
+            'Breadcrumbs' => ['className' => 'BootstrapUI.Breadcrumbs'],
+        ];
+
+        $this->helpers = array_merge($helpers, $this->helpers);
```

Comment:
Should we use assoc + operator? Or do we need to allow recursive merging? I think the first should suffice.

Reply:
I have intentionally used `array_merge()` to ensure that in the resultant array the keys of `$helpers` stay first. This ensures that if `$this->helpers` contained a helper which for e.g. uses `Html` helper in it's `initialize()` method then that's an instance of `BootstrapUI.HtmlHelper` and not core `HtmlHelper.